diff --git a/a.ts b/a.ts
--- a/a.ts
+++ b/a.ts
@@ -0,0 +1,2 @@
+export const a = 1;
+export const a2 = 2;
diff --git a/b.ts b/b.ts
--- a/b.ts
+++ b/b.ts
@@ -0,0 +1,2 @@
+export const b = 1;
+export const b2 = 2;
diff --git a/c.ts b/c.ts
--- a/c.ts
+++ b/c.ts
@@ -0,0 +1,2 @@
+export const c = 1;
+export const c2 = 2;
diff --git a/d.ts b/d.ts
--- a/d.ts
+++ b/d.ts
@@ -0,0 +1,2 @@
+export const d = 1;
+export const d2 = 2;
